newsplease/config/config.cfg

```diff
@@ -263,6 +263,13 @@ end_date = '2999-12-31 00:00:00'
 strict_mode = False
 
 
+[ContentFilter]
+
+filter_predicates = {'title': 'cloud computing',
+                     'text': 'google | amazon'}
```

Comment:
Can you add a brief comment to explain how this is to be used? E.g., that you can use the term filter on either the title or the text field.

Reply:
Also, can one use pure strings or also regex?

Reply:
Pure string without &, | and ( is supported. 
regex is not supported. I feel logic expression is easier to use comparing with regex. with nested logic expression, there will be rare case where regex is needed